steps/xcode-build-for-test/0.4.0/step.yml

```diff
@@ -0,0 +1,140 @@
+title: '[BETA] Xcode Build for testing for iOS'
+summary: Performs xcodebuild's build-for-testing action
+description: Performs xcodebuild's build-for-testing action, to build tests without
+  running them.
+website: https://github.com/bitrise-steplib/steps-xcode-build-for-test
+source_code_url: https://github.com/bitrise-steplib/steps-xcode-build-for-test
+support_url: https://github.com/bitrise-steplib/steps-xcode-build-for-test/issues
+published_at: 2019-10-30T12:17:28.8773762Z
+source:
+  git: https://github.com/bitrise-steplib/steps-xcode-build-for-test.git
+  commit: 9cb8316edaa24f76513c68dfc5892addb36d7f33
+project_type_tags:
+- ios
+- react-native
+- cordova
+- ionic
```

Comment:
I think this should be available for Flutter as well.
I've created a card for it. 🏃